core_backend/app/contents/config.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 # PGVector variables
-PGVECTOR_VECTOR_SIZE = os.environ.get("PGVECTOR_VECTOR_SIZE", "1536")
+PGVECTOR_VECTOR_SIZE = os.environ.get("PGVECTOR_VECTOR_SIZE", "1024")
```

Comment:
Is this for gte-large? 
We don't define this env var in our deployment, which uses openai embeddings. So if the deployment just uses default value it may break?

Reply:
Can we keep it as original value and add to the docs that we should update this env var to match whatever embeddings we use?

Also realized we should either 
1. never allow switching embedding models
2. or allow switching, but also have some way to trigger reindexing of all content to that embedding.. 😅 future ticket?

Reply:
For now I will update back to 1536

Reply:
That's good to know. Generally we need an easier way to update these environment variables for deployments & updating vector db index..